code-samples/eventing/bookstore-sample-app/db/README.md

```diff
@@ -2,6 +2,7 @@
 
 1. Database Schema
 2. Sample Data
+3. Deployment on K8s Cluster

```

Comment:
Currently the README.md is not detailed enough, you can take a look at these 2 PRs, and see how doc is organzied. Please follow the same template, and make sure the steps are clear and well-explained. Also make sure you explain the reason behind each step.

[Example 1: Sentiment Analysis Doc](https://github.com/knative/docs/blob/2e6a0cfe411c1660cd8ca9da3740da439f7cd379/code-samples/eventing/bookstore-sample-app/ML-sentiment-analysis/README.md)
[Example 2: Camel-K slack sink Doc](https://github.com/Leo6Leo/docs/blob/e577254e5570f786237977355c7bbb05cad7123e/code-samples/eventing/bookstore-sample-app/slack-sink/README.md)